frontend/src/contexts/userContext.tsx

```diff
@@ -24,6 +24,7 @@ export type UserWorksheets = {
     [worksheetNumber: number]: {
       crn: Crn;
       color: string;
+      hidden: boolean;
```

Comment:
thanks for the comments Sida! still getting better at react, super helpful tips

Reply:
Yeah this one is just because we have some weird type setup. I set this up so I have no one to complain to, but I also don't have the energy to make even more convoluted infra to make it safer...